src/spyglass/utils/dj_helper_fn.py

```diff
@@ -301,8 +303,16 @@ def fetch_nwb(query_expression, nwb_master, *attrs, **kwargs):
             # This also opens the file and stores the file object
             get_nwb_file(file_path)
 
+    # logging arg only if instanced table inherits Mixin
```

Comment:
maybe at some point this `inst` piece should be extracted into a helper, or maybe as `def is_spymixin(obj) -> bool:` - but i'm good with merging as-is